src/App.js

```diff
@@ -1,29 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable no-unused-vars */
+/* eslint-disable class-methods-use-this */
+import React from 'react';
+import Calculator from './components/Calculator';
```

Comment:
- [ ] The path of your component does not match. Please fix it to be able to proceed with the project 😉

Reply:
Hi, what do you mean by "path does not match"?

Reply:
You have a component called calculator.js and you are calling it Calculator.js that throws a path error.

Reply:
![image](https://user-images.githubusercontent.com/88060989/155421974-9b7e1cd2-def3-4f1b-8d85-0fd0785c6e97.png)
 actually it shows w capped C in my local